hashicorp/terraform-aws-consul
# Update tests

Continuation of https://github.com/hashicorp/terraform-aws-consul/pull/171

- Convert to go modules
- Pull in latest terratest version
- Make corresponding changes to circleci script to support go modules
- Add support for terratest_log_parser
- Update all packer templates to be compatible with `1.5.4+`

Note that tests are still unstable. Different tests are failing each time with a timeout waiting for consul to come up. I don't quite have time to dig in, but it appears to be intermittent.
